lesson01/scripts.js

```diff
@@ -0,0 +1,41 @@
+
+/* this is the random array */
+const array = [5, 18, 27, 30, 66];
+
+$(document).ready(function(){
+  /*show the original array in the html*/
+  $('#array').html(array.join(" "));
+
+/*this is the first input function -- adding a number to each number in the array to create a new array*/
+  $('.submit_one').click(function(){
+    let array_one = array.map(value => value + parseInt($('.input_one').val()));
+    $('#array_one').html(`Here are the numbers in the array with their new values: ${array_one.join(" ")}.`);
+  });
+
+/*this is the second input function -- finding numbers greater than the input value*/
+
+  $('.submit_two').click(function(){
+    let big_numbers = array.filter(value => value > $('.input_two').val());
+    $('#array_two').html(`Here are the numbers greater than the value you provided: ${big_numbers.join(" ")}.`);
+  });
+
+/*this is the third input function -- checking if a number is in the array*/
+
+  $('.submit_three').click(function(){
+    let input = +$('.input_three').val(); /*must include + or unary operator bc input returns string and includes checks only similar object types*/
+    let number_check = array.includes(input);
+    let number_check_output = number_check.toString();
+    $('#array_three').html(`The number you provided has been checked if it is in the array: ${number_check_output}.`);
```

Comment:
I like that you're using the + unary operator to convert input to a number; it allows for a much wider range of possible inputs. However, since `parseInt` is a more common approach to converting a string to an integer, you may want to make your comment slightly more specific, like "Using + or unary operator to convert string input to nuber; 'includes' checks only similar object types".

No need to convert the number back to a string to print it. Javascript is so easy going that you can just insert a numeric value in a string and it'll print to the page with no problems.

Reply:
Thanks, Cheri! This is great insight for future JS.